restapi/src/main/java/io/stargate/web/docsapi/dao/DocumentDB.java

```diff
@@ -151,6 +151,13 @@ public boolean maybeCreateTable(String keyspaceName, String tableName) {
       throw new ErrorCodeRuntimeException(ErrorCode.DATASTORE_KEYSPACE_DOES_NOT_EXIST, message);
     }
 
+    // if table exists, do nothing
+    Table table = ks.table(tableName);
```

Comment:
This check is done for performance reasons? Instead of round-tripping to the datastore?

Reply:
Yes, I guess so. This is called on every document write, so we don't want to go to the store every time.